hs_core/models.py

```diff
@@ -270,7 +270,7 @@ def update(cls, element_id, **kwargs):
         if 'description' in kwargs:
             party = cls.objects.get(id=element_id)
             if party.description is not None and kwargs['description'] is not None:
-                if len(party.description) > 0 and len(kwargs['description']) > 0:
+                if len(party.description.strip()) > 0 and len(kwargs['description'].strip()) > 0:
                     if party.description != kwargs['description']:
```

Comment:
```strip()``` is not needed on line 274 since at that point we already know that we are not comparing two empty strings.